utils/build/clean.js

```diff
@@ -1,6 +1,6 @@
-const { packages } = require("../list_packages");
+const { workspace } = require('../workspace');
 const path = require('path');
 const rimraf = require('rimraf');
-for (const packageDir of packages) {
-  rimraf.sync(path.join(packageDir, 'lib'));
-} 
\ No newline at end of file
+for (const pkg of workspace.packages()) {
+  rimraf.sync(path.join(pkg.path, 'lib'));
```

Comment:
Why isn't this `node utils/workspace.js --clean`?

Reply:
build scripts are left as-is for now.